pkgs/by-name/fi/figurine/package.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+}:
+
+let
+  version = "1.3.0";
+in
+buildGoModule rec {
+  pname = "figurine";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "arsham";
+    repo = "figurine";
+    rev = "v${version}";
+    hash = "sha256-1q6Y7oEntd823nWosMcKXi6c3iWsBTxPnSH4tR6+XYs=";
+  };
+
+  vendorHash = "sha256-mLdAaYkQH2RHcZft27rDW1AoFCWKiUZhh2F0DpqZELw=";
+
+  ldflags = [
+      "-s"
+      "-w"
+      "-X main.version=${version}"
+      "-X main.currentSha=${src.rev}"
+  ];
```

```suggestion
  ldflags = [
    "-s"
    "-w"
    "-X main.version=${version}"
    "-X main.currentSha=${src.rev}"
  ];
```

Indentation is wrong.

---

sure, yup.